daprdocs/content/en/operations/configuration/preview-features.md

```diff
@@ -0,0 +1,94 @@
+---
+type: docs
+title: "How-To: Enable preview features"
+linkTitle: "Preview Features"
+weight: 7000
+description: "How to specify and enable preview features"
+---
+
+## Overview
+Some features in dapr are considered experimental when they are first released. These features require explicit opt-in in order to be used. The opt-in is specified in dapr's configuration.
+
+Currently, preview features are enabled on a per application basis. A global scope may be introduced in the future should there be a use case for it.
+
+### Current Preview Features
+Below is a list of existing preview features:
+- Actor Reentrancy
```

Comment:
Do we have a doc on actor reentrancy (what it is, how to use it)?

Reply:
If so we should link to it here, if not we should add a page

Reply:
Looks like an issue for it was missed. I'll tackle that today and include the reference between the two in whichever PR is approved/merged later.

https://github.com/dapr/docs/issues/1482

Reply:
We'll get this in within the other PR so I can merge this and the other build will complete